components/CreateOrganizationForm.js

```diff
@@ -1,289 +1,387 @@
+  slugLabel: { id: 'createOrg.form.slugLabel', defaultMessage: 'What URL would you like?' },
+  websiteLabel: { id: 'createOrg.form.webstiteLabel', defaultMessage: "What's your Organization's website" },
+  suggestedLabel: { id: 'createOrg.form.suggestedLabel', defaultMessage: 'Suggested' },
+  descriptionLabel: {
+    id: 'createOrg.form.descriptionLabel',
+    defaultMessage: 'What does your organization do?',
+  },
+  descriptionHint: {
+    id: 'createOrg.form.descriptionHint',
+    defaultMessage: 'Write a short description of your Organization (150 characters max)',
+  },
+  descriptionPlaceholder: {
+    id: 'create.org.placeholder',
+    defaultMessage: 'Making the world a better place',
+  },
+  errorName: {
+    id: 'createOrg.form.error.name',
+    defaultMessage: 'Please use fewer than 50 characters',
+  },
+  errorDescription: {
+    id: 'createOrg.form.error.description',
+    defaultMessage: 'Please use fewer than 160 characters',
+  },
+  errorWebsite: {
+    id: 'createOrg.form.error.website',
+    defaultMessage: 'Enter valid website format',
+  },
+});
 
-    this.messages = defineMessages({
-      'name.label': { id: 'Fields.name', defaultMessage: 'Name' },
-      'description.label': {
-        id: 'collective.description.label',
-        defaultMessage: 'Short description',
-      },
-      'website.label': {
-        id: 'Fields.website',
-        defaultMessage: 'Website',
-      },
-      'website.description': {
-        id: 'collective.website.description',
-        defaultMessage: 'Enter the URL of your website or Facebook Page',
-      },
-      'tags.label': { id: 'Tags', defaultMessage: 'Tags' },
-      'tags.description': {
-        id: 'collective.tags.description',
-        defaultMessage: 'Tags helps people discover your collective',
-      },
-      'tos.label': {
-        id: 'collective.tos.label',
-        defaultMessage: 'Terms of Service',
-      },
-    });
+const placeholders = {
+  name: { id: 'placeholder.name', defaultMessage: 'i.e. Salesforce, Airbnb' },
+  slug: { id: 'placeholder.slug', defaultMessage: 'Airbnb' },
+  description: { id: 'placeholderdescription', defaultMessage: 'Making a world a better place' },
+  website: { id: 'placeholder.website', defaultMessage: 'www.airbnb.com' },
+  username: { id: 'placeholder.username', defaultMessage: 'User name' },
+};
 
-    collective.backgroundImage = collective.backgroundImage || defaultBackgroundImage[collective.type];
-
-    this.masterKey = '';
-    this.state = { modified: false, collective };
-    this.defineFields();
-    this.addLabels();
-  }
+function CreateOrganizationForm(props) {
+  const { intl, error, loading, LoggedInUser, onSubmit, updateAdmins } = props;
+  const [authorization, setAuthorization] = useState('');
+  const [admins, setAdmins] = useState([]);
+  const initialValues = {
+    name: '',
+    slug: '',
+    description: '',
+    website: '',
+    coAdmin: '',
+  };
+  const validate = values => {
```

Comment:
Validation that the website is a URL happened on the server side but not the frontend, we want to validate it before we send it off to the server in the graphql mutation.

<img width="885" alt="Screenshot 2020-10-07 at 11 38 21" src="https://user-images.githubusercontent.com/37520401/95321083-70769880-0892-11eb-8420-e46e01dfc5da.png">

Reply:
Done